ci/setup_dcos-cli.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -eu -o pipefail
+
+mkdir ./bin
+curl -o ./bin/dcos https://downloads.dcos.io/binaries/cli/linux/x86-64/0.8.0/dcos
+chmod +x ./bin/dcos
+
+export PATH=./bin/:$PATH
+echo "##teamcity[setParameter name='env.PATH' value='${PATH}']"
+echo PATH: "$PATH"
+
+mkdir ./dcos_dir
+DCOS_DIR=$(pwd)/dcos_dir
+export DCOS_DIR
+echo "##teamcity[setParameter name='env.DCOS_DIR' value='${DCOS_DIR}']"
+
+readonly USERNAME=bootstrapuser
```

Comment:
As Sam says, these should be injected from TeamCity.

Reply:
done